src/cli.js

```diff
@@ -34,4 +34,6 @@ module.exports = async ({ customTemplatePath, useDefaultAnswers }) => {
   await readme.writeReadme(readmeContent)
 
   utils.showEndMessage()
+
+  yargs.exit(0)
```

Comment:
Doing an exit is a bit rough. Actually I found the bug, it's a yargs regression, the solution is just to fix the version to 14.0.0